e2e/go.mod

```diff
@@ -169,4 +169,8 @@ require (
 )
 
 // TODO: using version v1.0.0 causes a build failure. This is the previous version which compiles successfully.
-replace github.com/ChainSafe/go-schnorrkel => github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
+replace (
+	github.com/ChainSafe/go-schnorrkel => github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
+	github.com/ChainSafe/go-schnorrkel/1 => github.com/ChainSafe/go-schnorrkel v1.0.0
+	github.com/cosmos/ibc-go/v5 => ../
```

Comment:
I'm a bit hesitant to pin this.

Currently, `ibctest` is pinned to v5, if we update to v6, the tests will fail due to a duplicate message registration with multiple `ibc-go` imports.

We're explicitly keeping it separate right now, but I think having this as a commented line at the bottom of the file would be very useful for development, i.e.

```
// uncomment to use the local version of ibc-go
// replace github.com/cosmos/ibc-go/v5 => ../`
```

Reply:
OK, Yeah, it's better.